dev/breeze/tests/test_selective_checks.py

```diff
@@ -341,7 +341,7 @@ def assert_outputs_are_printed(expected_outputs: dict[str, str], stderr: str):
         ),
         (
             pytest.param(
-                ("providers/tests/apache/beam/file.py",),
+                ("providers/apache/beam/tests/file.py",),
```

Comment:
@potiuk @eladkal Shall I remove this `pytest.param` config altogether? I am not able to figure out why only `apache.beam common.compat google` are in the `selected-providers-list-as-string`.

Reply:
it's becuase the providers are importing each other. Selecitve checks will choose all the "related" providers (i.e. those importing the changed provider) when some provider changes.

Reply:
Thanks!